src/datasets/argument_relation_detection_dataset.py

```diff
@@ -13,28 +13,34 @@ def __init__(
         self,
         json_path: Path,
         tokenizer: PreTrainedTokenizer,
+        test_mode: bool = False,
     ):
         with open(json_path, "r") as f:
             data = json.load(f)
-        self.texts: list[tuple[str, str]] = [(sample[0], sample[1]) for sample in data]
-        self.labels: list[int] = [sample[2] for sample in data]
+        self.texts: list[tuple[str, str]] = [(sample[1], sample[2]) for sample in data]
+        self.labels: list[int] = [sample[3] for sample in data] if not test_mode else None
```

Comment:
labels aren't

list[int]

anymore, they're 

list[int] | None

this type annotation means that it can be either list[int] or None. I like type annotations so I think it's good to adjust this.

Also this problem is caught by the mypy typechecker if you run

mypy .

in the main directory of the project (setup.sh includes setup for mypy, it should work).